src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
+
+    subtract(num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide(num) {
+      this.result /= num;
```

Comment:
There is a potential division by zero issue in the `divide` method. Consider adding a check to prevent division by zero, which could lead to unexpected results or errors.